dist/index.html

```diff
@@ -82,12 +82,18 @@ <h1 class = "title">What's Cookin?</h1>
     <section class="all-recipes-display hidden">
       <!-- DOM elements go here -->
     </section>
+    <div class="save-recipe-button hidden">
+      <button class="save-recipe-btn">Save Recipe</button>
+    </div>
     <section class="single-recipe-display hidden">
-      <h2>Recipe Name</h2>
-      <img id="front-recipe-1" src="https://spoonacular.com/recipeImages/741603-556x370.jpeg" class="recipe" alt='recipe-title'>
-      <p class="total-cost-box">The total cost to make this recipe is: $1 million</p>
-      <p class="ingredient-box">Ingredients Box</p>
-      <p class="instruction-box"></p>
+      <div class="recipe-name-section">
+        <h2 class="recipe-name">RecipeName</h2>
+      </div>
+      <!-- <section class="recipe-info">
+      </section> -->
+      <!-- DOM elements go here -->
+    </section>
```

Comment:
I like how this section is divided. Will make it much easier for styling later on!